python/lsst/ts/xml/data/sal_interfaces/MTCamera/MTCamera_Telemetry.xml

```diff
@@ -8859,14 +8881,21 @@
     <Subsystem>MTCamera</Subsystem>
     <EFDB_Topic>MTCamera_fcs_Counters_Autochanger_Trending</EFDB_Topic>
     <Description>subsystem MTCamera class MTCamera_fcs_Counters_Autochanger_Trending for category Trending</Description>
-    <!--CCS: Dictionary_Checksum: 1345145492L -->
+    <!--CCS: Dictionary_Checksum: 1395154614L -->
     <item>
       <EFDB_Name>autochangertrucks_ALIGN_FOLLOWER</EFDB_Name>
       <Description>Counter for the action ALIGN_FOLLOWER</Description>
       <IDL_Type>double</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
+    <item>
+      <EFDB_Name>autochangertrucks_MOVE_DRIVER_TRUCK_ALONE</EFDB_Name>
+      <Description>Counter for the action MOVE_DRIVER_TRUCK_ALONE</Description>
+      <IDL_Type>double</IDL_Type>
```

Comment:
Lots of these parameters are counters. Does a double make sense here? Are we going to get fractional counts?

Reply:
Good point, we also wanted to change this but it is something specific to the Counter implementation from the toolkit. It has been this way for some time now so it would have been a big change.